index.html

```diff
@@ -61,67 +61,31 @@
           <div class="carousel-food-spin js-carousel-food-spin" data-speed="800">
             <ul class="carousel-food-spin__dishs js-carousel-food-spin-dishs" aria-label="available menus">
               <li class="carousel-food-spin__dish">
-                <div class="carousel-food-spin__main js-carousel-foord-spin-dish">
-                  <a href="" aria-label="click to see the selected dish">
-                    <img class="carousel-food-spin__img" src="image/food1.png" alt="dish salad">
-                  </a>
-                </div>
+                <dishe-element name="food1.png" data-price="32" data-title="Green Goddess Chicken Salad" data-description="'It Is A Non Vegetarian Salad Which Consists Of The Green Goddess Dressing Mixed With Chicken, Peppers, Olives And Celery."></dishe-element>
```

Comment:
Lo que veo aqui es el uso de los atributos. No tienen que ser `data-`, puedes ponerle nombres cualquiera.
```suggestion
                <dishe-element image="food1.png" price="32" name="Green Goddess Chicken Salad" description="'It Is A Non Vegetarian Salad Which Consists Of The Green Goddess Dressing Mixed With Chicken, Peppers, Olives And Celery."></dishe-element>
```